File: 2026A/Unidad 2/BST/Node.js
class Node<T> {
    int key;           // Clave para ordenar (por ejemplo, cédula)
    T data;            // Dato genérico almacenado
    Node<T> left;      // Referencia al hijo izquierdo
    Node<T> right;     // Referencia al hijo derecho

    // Constructor del nodo
    public Node(int key, T data) {
        this.key = key;
        this.data = data;
        this.left = null;
        this.right = null;
    }
}

File: 2026B/Unidad 1/Node.java
/**
 * Nodo de la lista simplemente enlazada.
 * Guarda el dato y la referencia al siguiente nodo.
 */
public class Node<E> {
    E data;
    Node<E> next;

    public Node(E data) {
        this.data = data;
        this.next = null;
    }
}
